sql-gremlin/src/test/java/org/twilmes/sql/gremlin/adapter/GremlinSqlAdvancedSelectTest.java

```diff
@@ -147,7 +147,7 @@ public void testWhere() throws SQLException {
                 rows(r("Tom", 35), r("Juanita", 50)));
     }
 
-    // TODO: Support NOT on operators
+    // TODO: Support where NOT on operators
```

Comment:
Shall we remove TODO, since all Disabled tests are actually a todo. The additional comment might be unnecessary?